docs/scripts/youtube-embeds.js

```diff
@@ -1,18 +1,32 @@
 $(document).ready(function () {
     $("img").each(function(idx, img) {
-        var alt = img.alt.toLowerCase();
+        const alt = img.alt.toLowerCase();
         if (!(alt == "yt" || alt == "youtube")) {
             return;
         }
 
-        // Regex from https://stackoverflow.com/a/37704433
-        var id = img.src.match(/^((?:https?:)?\/\/)?((?:www|m)\.)?((?:youtube\.com|youtu.be))(\/(?:[\w\-]+\?v=|embed\/|v\/)?)([\w\-]+)(\S+)?$/)[5];
+        // Original regex taken from https://stackoverflow.com/a/37704433
+        const src = img.getAttribute('src'),
```

Comment:
Why not just `img.src`?

Reply:
The browser would convert it to an absolute path, so inputs with just the ID fail.